apps/frontend-manage/src/components/evaluation/elements/FTEvaluation.tsx

```diff
@@ -18,27 +25,56 @@ function FTEvaluation({
   chartType,
   showSolution,
 }: FTEvaluationProps) {
+  const [isCollapsed, setIsCollapsed] = useLocalStorage(
```

Comment:
same

Reply:
used useState instead now to be able to hide sidebar because else part of content is still visible in collapsed state